frontend.java

```diff
@@ -0,0 +1,540 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
```

Comment:
You should have have multiple classes when implementing this project.  Having the following classes could have added in the design and implementation.

- Accounts
- Transaction
- Frontend